routingtable/adjRIBIn/adj_rib_in.go

```diff
@@ -20,16 +20,18 @@ type AdjRIBIn struct {
 	contributingASNs *routingtable.ContributingASNs
 	routerID         uint32
 	clusterID        uint32
+	addPathRX        bool
 }
 
 // New creates a new Adjacency RIB In
-func New(exportFilter *filter.Filter, contributingASNs *routingtable.ContributingASNs, routerID uint32, clusterID uint32) *AdjRIBIn {
+func New(exportFilter *filter.Filter, contributingASNs *routingtable.ContributingASNs, routerID uint32, clusterID uint32, addPathRX bool) *AdjRIBIn {
```

Comment:
Because of this change the test fails with:
not enough arguments in call to New
        have (*filter.Filter, *routingtable.ContributingASNs, uint32, uint32)
        want (*filter.Filter, *routingtable.ContributingASNs, uint32, uint32, bool)